src/pages/Login/Signup.components.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import './Signup.styles.scss';
+
+class SignUp extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            name: '',
+            email: '',
+            password: '',
+            type: 'password',
+        };
+    }
+
+    showHide = (e) => {
+        e.stopPropagation();
```

Comment:
Could you explain to me why did you used `stopPropagation();`?

Reply:
it is wrong I mean I wanted to use preventDefault but did type stopPropogation